src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+public class Figure {
```

Comment:
this class should be abstract and should contain only common fields for all figures. The common field is only `color`